projects/plugins/jetpack/modules/subscriptions/subscribe-modal/subscribe-modal.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * Subscribe modal popup feature
+ *
+ * @package automattic/jetpack-mu-wpcom
+ * @since 2.2.1
```

Comment:
Package name needs updating/removing

Needs some `$` variable here that gets automatically replaced with right version later I think.

Reply:
+1. That would give us

```suggestion
 * @since $$next-version$$
```

For more information:
https://github.com/Automattic/jetpack/blob/2aac5857f66d07bfda1e08da7db299b9aaf63db9/projects/packages/README.md#package-version-annotations

Reply:
I've updated this!